bin/reportdaily.py

```diff
@@ -99,10 +314,10 @@ def cmd_export(args):
 
 
 def parsecli(cliargs=None) -> argparse.Namespace:
-    """Parse CLI with :class:`argparse.ArgumentParser` and return parsed result
+    """Parse CLI with: class: `argparse.ArgumentParser` and return parsed result
```

Comment:
The space between `:class:` and the class name is essential. It's actually a macro which doesn't allow any whitespaces in between:

```suggestion
    """Parse CLI with: class:`argparse.ArgumentParser` and return parsed result
```